electronics_lib/PowerConditioning.py

```diff
@@ -263,3 +264,136 @@ def connected_from(self, gnd: Optional[Port[VoltageLink]] = None, pwr_hi: Option
     if pwr_lo is not None:
       cast(Block, builder.get_enclosing_block()).connect(pwr_lo, self.pwr_lo)
     return self
+
+
+class PmosReverseProtection(PowerConditioner, KiCadSchematicBlock, Block):
+  """
+  - This circuit uses a PMOS as a switch, which turns on when the power supply is correctly connected,
```

Comment:
Perhaps the first sentence should focus on what this does instead of how it's implemented?
e.g. this block protects the load against reverse voltage (e.g. a battery connected backwords) by using a MOS switch that does not incur a voltage drop like the simpler diode circuit